ep_testing/tests/base.py

```diff
@@ -1,7 +1,14 @@
+from os import chdir
+from tempfile import mkdtemp
+
+
 class BaseTest:
 
     def __init__(self):
         self.verbose = False
+        temp_dir = mkdtemp()
+        print('{Sandbox Dir: \"' + temp_dir + '\"} ', end='')
+        chdir(temp_dir)
```

Comment:
Have the base test create a sandbox directory...I think I need to clean this prior to merge.